file/configfiles.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# Get various config files
+
+import logging
+import os
+
+from glob import glob
+
+from file.base import FileCollecting
+from utils import util
+from utils import fileopt
+
+
+class InsightConfigFiles(FileCollecting):
+    def save_sysconf(self):
+        cmd = ["sysctl", "-a"]
+        path_limit_file = "/etc/security/limits.conf"
+
+        # save limits.conf
+        self.save_to_dir(srcfile=path_limit_file)
+
+        # save `sysctl -a`
+        stdout, stderr = util.run_cmd(cmd)
+        if stdout:
+            fileopt.write_file(os.path.join(
+                self.outdir, "sysctl.conf"), stdout)
+        if stderr:
+            fileopt.write_file(os.path.join(self.outdir, "sysctl.err"), stderr)
+
+    def find_tidb_configfiles(self, cmdline=None):
+        cmd_opts = util.parse_cmdline(cmdline)
+        # TODO: support relative path, this require `collector` to output cwd of process
+        try:
+            return cmd_opts["config"]
+        except KeyError:
+            return None
+        return
```

Comment:
why sometimes return None and sometimes return nothing, the others return config.

Reply:
made it uniform